metastore/scripts/upgrade/hive/hive-schema-3.0.0.hive.sql

```diff
@@ -0,0 +1,1334 @@
+-- HIVE system db
+
+CREATE DATABASE SYS;
```

Comment:
Gunther, are there any reasons that you create the tables in sys using JdbcStrageHandler instead of creating views on top of their corresponding tables in HMS?